airflow/www/security.py

```diff
@@ -409,6 +474,20 @@ def has_access(self, permission, resource, user=None) -> bool:
 
         return has_access
 
+    def _has_access(self, user, action, resource) -> bool:
```

Comment:
I think "action_name" and "resource_name" should be here as well.

Reply:
Agreed